src/constants.ts

```diff
@@ -23,16 +23,27 @@ export const SERVER_DESCRIPTION_CHANGED = 'serverDescriptionChanged' as const;
 export const TOPOLOGY_OPENING = 'topologyOpening' as const;
 export const TOPOLOGY_CLOSED = 'topologyClosed' as const;
 export const TOPOLOGY_DESCRIPTION_CHANGED = 'topologyDescriptionChanged' as const;
+/** @public */
```

Comment:
This was done since the the `name` property that we have on our Command Events and ConnectionPool Events is public so we need to have these constants exported. I'm open to doing this in a more elegant way.

Reply:
Well one option is to make the `name` property on the command events internal.  There's no reason it needs to be public, right?  It's only used by the logger.